apps/web/src/app/dashboard/[name]/page.tsx

```diff
@@ -1,13 +1,17 @@
-"use client";
+import { headers } from "next/headers";
+import { redirect } from "next/navigation";
 
-import { usePathname, useRouter } from "next/navigation";
-import { useEffect } from "react";
+import { api } from "~/trpc/server";
 
-export default function OrganizationView() {
-  const router = useRouter();
-  const pathname = usePathname();
-  useEffect(() => {
-    router.push(`${pathname}/overview`);
-  });
-  return <></>;
+export default async function Organization() {
+  const heads = headers();
+  const orgFromPath = heads.get("next-url")?.split("/").at(2)!;
+  const organizations = await api.organization.getOwnOrganizations.query();
+  const orgNames = organizations.map((org) => org.name!);
+
+  if (orgNames.includes(orgFromPath)) {
+    redirect(`/dashboard/${orgFromPath}/overview`);
+  } else {
+    redirect("/dashboard");
+  }
```

Comment:
This also should be changed in when addressing [this](https://github.com/users/hubcio2115/projects/3/views/1?query=is%3Aopen+sort%3Aupdated-desc&pane=issue&itemId=60319390)